app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1115,6 +1120,44 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         startActivity(AddWidgetInstructionsActivity.intent(context), options)
     }
 
+    override fun onDismissDialog() {
+        val configuration = viewModel.ctaViewState.value?.cta ?: return
+        if (configuration is DaxDialogCta.DaxTrackersBlockedCta) {
+            animatorHelper.finishTrackerAnimation(omnibarViews(), networksContainer)
+        }
+        viewModel.onUserDismissedCta(configuration)
+    }
+
+    override fun onHideClick() {
+        context?.let {
+            val configuration = viewModel.ctaViewState.value?.cta ?: return
```

Comment:
same here, this could be moved to the `ViewModel`

Reply:
Done!